compiler/modules/types/list.bal

```diff
@@ -22,6 +22,21 @@ type ListConjunction record {|
     ListConjunction? next;
 |};
 
+public type ListMemberTypes [Range[], SemType[]];
+
+function listMemberTypesUnion(ListMemberTypes mt1, ListMemberTypes mt2) returns ListMemberTypes {
```

Comment:
Since this is non-public, it should be further down the file (just before combineRanges).